content/docs/glossary/common-rkvst-terms/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Common RKVST Terms"
+description: ""
+lead: ""
+date: 2022-10-19T07:39:44-07:00
+lastmod: 2022-10-19T07:39:44-07:00
+draft: false
+images: []
+menu: 
+  docs:
+    parent: "glossary"
+weight: 51
+toc: true
+---
+
+Select a term for more information.
+
+| **Term**            | **Definition**                                                                                                       |
+|---------------------|----------------------------------------------------------------------------------------------------------------------|
+| [ABAC](https://docs.rkvst.com/docs/rkvst-basics/managing-access-to-an-asset-with-abac/)                | Attribute-Based Access Control; policy that allows you to grant fine-grain access to members of your tenancy         |
+| [access policy](https://docs.rkvst.com/docs/overview/core-concepts/#access-policies)       | grants chosen asset and event access to stakeholders                                                                 |
+| [app registration](https://docs.rkvst.com/docs/rkvst-basics/getting-access-tokens-using-app-registrations/#creating-an-app-registration)    | client ID and client secret credentials that are used to access the RKVST API                                        |
+| [asset](https://docs.rkvst.com/docs/overview/core-concepts/#assets)               | an RKVST asset has a collection of attributes that describes its current state and a complete life history of events |
```

Comment:
Can we go even more basic with this? "an RKVST asset is an entry in your tenancy which has a collection of attributes that describes its current state and a complete life history of events"

Reply:
Yep, changed!